homework/12_V/15_Yordan/Homework1/test_get_intersections.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from get_intersections import *
+
+def test_intersections():
+    print(get_intersections(0, 0, 10, 5, 5, 10))
+    assert get_intersections(0, 0, 5, 2, 2, 5) == (4.391164991562634, -2.3911649915626336, -2.3911649915626336, 4.391164991562634)
+    assert get_intersections(0, 0, 3, 0, 0, 5) == None

```

Comment:
The proper way of comparing for None is `get_intersections(...) is None` not using `== None`